mkosi.uki-profiles/10-live.conf

```diff
@@ -17,5 +17,6 @@ Cmdline=
         systemd.journald.max_level_console=warning
         rw
         audit=0
+        systemd.image_policy=esp=unprotected:xbootldr=unprotected+unused+absent:=ignore
```

Comment:
No verity on usr here?

Reply:
for usr/root the policy is not enforced currently, because we do not use the dissection logic for it currently. we really should though. i guess htat's something really to fix though